### PR TITLE
Add trap flow counter support

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
#### What I did

Add trap flow counter CLI support. See HLD: https://github.com/Azure/SONiC/pull/858

#### How I did it

Add new command:

- counterpoll flowcnt-trap enable/disable
- counterpoll flowcnt-trap interval
- show flowcnt-trap stats
- sonic-clear flowcnt-trap
- Extend command: counterpoll show

#### How to verify it

Unit test/Manual test

### CODE REVIEW

**Comment by Junchao-Mellanox** [comment]
Hi @qiluo-msft , could you please suggest a reviewer from MSFT side? Thanks!

**Comment by prsunny** [comment]
@dgsudharsan, can you review this as well?

File: scripts/flow_counters_stat [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+            if ns not in old_data:
+                continue
+            old_stats = old_data[ns]
+            for name, values in stats.items():
+                if name not in old_stats:
+                    continue
+
+                old_values = old_stats[name]
+                if values[-1] != old_values[-1]:
+                    # Counter OID not equal means the trap was removed and added again. Removing a trap would cause
```
Comment (dgsudharsan) [inline scripts/flow_counters_stat:222]: This is a good logic to handle. However, what if OID returned is the same as before delete? Should we rather not check if the diff is negative and consider it as a case where trap got deleted?

**Review by dgsudharsan — CHANGES_REQUESTED** [review]

**Review by dgsudharsan — APPROVED** [review]

**Review by keboliu — APPROVED** [review]

**Review by prsunny — APPROVED** [review]

**Pull request merged**